tests/rules/test_CrossAccountTrustRule.py

```diff
@@ -152,9 +154,15 @@ def test_service_is_not_blocked(mock_logger, template_valid_with_service):
     assert len(result.failed_rules) == 0
     assert len(result.failed_monitored_rules) == 0
 
-    mock_logger.assert_called_with(
-        "Not adding CrossAccountTrustRule failure in LambdaRole because no AWS Account ID was found in the config."
-    )
```

Comment:
Notice this test case / assertion has been removed, but the logger can still be triggered in the code. Has every branch of this IF statement been tested in the unit tests here: https://github.com/Skyscanner/cfripper/blob/rename_files_snake_case/cfripper/rules/cross_account_trust.py#L53-L73

Reply:
Yep, but this is not the case where the logger it should act. This tests checks that an AWS service is not a cross-account relationship. So no error should be raised and neither no log.
The previous if does this check.